### ISSUE TITLE
bug

### ISSUE THREAD

**Issue by HenzelMoras** [hugo-toha/toha#273]
## It does not get displayed

![nabar prob](https://user-images.githubusercontent.com/60033549/113189746-28e92c00-9279-11eb-89b9-3c74ddb1dd3c.png)

## Possible Solution

within layouts/partials/navigators/navbar.html
```
{{ if gt $sectionCount 4 }}
          <li class="nav-item dropdown">
            <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">{{i18n "more" }}</a>
            <div class="dropdown-menu" aria-labelledby="navbarDropdown">
                {{ $sectionCount := 0 }}
                {{ range sort $sections "section.weight" }}
                  {{ if and (.section.enable) (.section.showOnNavbar) }}
                    {{ $sectionCount  = add $sectionCount 1}}
                    {{ if gt $sectionCount 4 }}
                      <a class="dropdown-item" href="#{{ partial "helpers/get-section-id.html" . }}">{{ .section.name }}</a>
                    {{ end }}
                  {{ end }}
                {{- end }}
```
## Steps to Reproduce

1. just place the weight of the section to 5 

### fix

![navbar-fix](https://user-images.githubusercontent.com/60033549/113190211-b0369f80-9279-11eb-8434-864d4243073c.png)

### Browsers Affected

- [x] Chrome
- [x] Firefox

**Comment by HenzelMoras** [comment]
@hossainemruz  should i send a pr with redirection fixs #272

**Comment by hossainemruz** [comment]
Hm.. I see the issue. Please send a PR.

**Comment by hossainemruz** [comment]
Please send separate PR for the two issues

**Comment by HenzelMoras** [comment]
> Please send separate PR for the two issues

**Comment by hossainemruz** [comment]
Fixed in #279
